tutorial-server/notebooks/CactusTutorial.ipynb

```diff
@@ -101,7 +101,21 @@
     "os.environ[\"OMPI_MCA_rmaps_base_oversubscribe\"] = \"true\"\n",
     "os.environ[\"OMPI_MCA_hwloc_base_binding_policy\"] = \"none\"\n",
     "os.environ[\"OMPI_MCA_btl_vader_single_copy_mechanism\"] = \"none\"\n",
-    "import scrolldown"
+    "#import scrolldown\n",
+    "\n",
+    "# We are going to install kuibit, a Python package to post-process Cactus simulations.\n",
+    "# We will install kuibit inside the Cactus directory. The main reason for this is to\n",
+    "# have a make easier to uninstall kuibit (you can just remove the Cactus folder). \n",
+    "os.environ[\"PYTHONUSERBASE\"] = os.path.join(os.environ[\"HOME\"], \"/Cactus\")\n",
```

Comment:
And... it does not even suffice to set PYTHONUSERBASE after Python has started. One needs to force it to reread the env variable, and this would have to happen _after_ `pip3 install kuibit` since Python checks if PYTHONUSERBASE contains a correct site-packages structure before adding it to `sys.path`. :-( 

Note that the same issue exists with not setting PYTHONUSERBASE at all. Namely this:  
```
mv ~/.local ~/\_local.bak  
python3 -c 'import os;os.system("pip3 install kuibit==1.3.3");import kuibit'  
```  
fails since at the time Python starts there is no `~/.local`. A second:  
```  
python3 -c 'import os;os.system("pip3 install kuibit==1.3.3");import kuibit'  
```  
succeeds since at that point `pip3` has created the expected directories.  
  
So: needs work, in all cases.

An ugly way to make this work is have a cell:
```
import site
site.USER_SITE = None
site.USER_BASE = None
site.main()
```
after running `pip3` which will force `site` to reinterpret `PYTHONUSERBASE` at a point where the looked for directories are present.

Reply:
Great catch!

It was working on my computer because I already have kuibit system-wide.

I added a small function to reload `PYTHONUSERBASE`. It should be good now.